src/index.js

```diff
@@ -15,19 +15,19 @@ import './i18n';
 import * as serviceWorker from './serviceWorker';
 import Loading from './loading';
 
+import 'react-toggle/style.css';
+
 Sentry.init({ dsn: 'https://66e6b4dc3d59463fa34272abcb5da6b1@sentry.virtuos.uos.de/4' });
 
 const App = lazy(() => import('./App'));
 
 ReactDOM.render(
-  <StrictMode>
-    <ThemeProvider theme={theme}>
-      <Global styles={GlobalStyle} />
-      <Suspense fallback={<Loading />}>
-        <App />
-      </Suspense>
-    </ThemeProvider>
-  </StrictMode>,
+  <ThemeProvider theme={theme}>
+    <Global styles={GlobalStyle} />
+    <Suspense fallback={<Loading />}>
+      <App />
+    </Suspense>
+  </ThemeProvider>,
```

Comment:
Because `react-transition-group` is currently provoking lots of warnings in strict mode. This makes debugging very annoying. As soon as they pull their stuff together we can enable strict mode again.